app/frontend/src/features/profile/ProfileTagInput.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { Meta, Story } from "@storybook/react/types-6-0";
+import * as React from "react";
+
+import ProfileTagInput from "./ProfileTagInput";
+
+export default {
+  title: "Profile/ProfileTagInput",
+  component: ProfileTagInput,
+  argTypes: {
+    onChange: { action: "Change value through react-hook-form" },
+  },
+} as Meta;
+
+const Template: Story<any> = (args) => <ProfileTagInput {...args} />;
```

Comment:
The component isn't updating because nothing is passed to `onChange`. I think something like this would work?:

```suggestion
const Template: Story<any> = (args) => {
  const [value, setValue] = setState(args.value);
  return <ProfileTagInput {...args} value={value} onChange={setValue}/>;
}
```